kategory/src/main/kotlin/kategory/instances/EitherMonad.kt

```diff
@@ -1,6 +1,6 @@
 package kategory
 
-class EitherMonad<L> : Monad<EitherF<L>> {
+open class EitherMonad<L> : Monad<EitherF<L>> {
```

Comment:
Sounds good, we can do a pass to redo all of them if this makes sense. Let's discuss by chat.